Content.IntegrationTests/Tests/PostMapInitTest.cs

```diff
@@ -97,6 +97,7 @@ public sealed class PostMapInitTest
             "Saltern",
             "Reach",
             "Oasis",
+			"Oasis2", //Modification - Carpmosia
```

Comment:
```suggestion
            "Oasis2", //Modification - Carpmosia
```
AFAIK all the files are indented with spaces, not tabs